packages/components/src/local/form-elements/message-creator/index.tsx

```diff
@@ -113,153 +108,30 @@ const MessageCreator: React.FC<PropTypes> = ({
 
   const onPopupConfirm = (event: MouseEvent<HTMLButtonElement>): void => {
     setConfirmIsOpen(false)
+    setClearName(messageOption)
     clearCachedCreatorMessage && clearCachedCreatorMessage([privatMessageOption, messageOption, UNSENT_SELECT_BY_DEFAULT_VALUE])
-    editor && editor.destroy()
     onCancel && onCancel(event)
   }
 
-  const destroyEditor = (editorObject: Editor | null): void => {
-    if (editorObject && (editorObject.ready || !editorObject.destroyed)) { editorObject.destroy() }
-  }
-
   useEffect(() => {
-    if (!schema) {
-      destroyEditor(editor)
-    }
+    const privateValues: string | undefined = getcachedCreatorMessageValue && getcachedCreatorMessageValue(privatMessageOption)
+    setPrivateValue(privateValues)
+    
     if (schema && (!selectedSchema || selectedSchema.title !== schema.title)) {
-      destroyEditor(editor)
       setSelectedSchema(schema)
     }
 
-    if (schema && schema.type) {
-      if (editor && (editor.ready || !editor.destroyed)) return
-      createEditor(schema)
-    }
-
-    return (): void => {
-      destroyEditor(editor)
-    }
   }, [schema, messageOption, confirmIsOpen])
 
-  useEffect(() => {
-    const formValue = getcachedCreatorMessageValue && getcachedCreatorMessageValue(messageOption)
-    const privateValue = getcachedCreatorMessageValue && getcachedCreatorMessageValue(privatMessageOption)
-    setPrivateValue(privateValue)
-
-    const valueTimer = setTimeout(() => {
-      if (formValue) return editor && editor.setValue(formValue)
-    }, 10)
-
-    return (): void => clearTimeout(valueTimer)
-  }, [editor])
-
-  /**
-   * helper function to for validation Datetime or Date or Time props of json
-   */
-  const configCommonValidation = (
```

Comment:
@lilitkarapetyan  - it looks like we're losing a block of functionality here.    I guess we have to move these custom editors, and custom validators to `JSON-Editor`.  Aah, unless they are there already :-)

Reply:
I will work on it